### PR TITLE
Update angular

### PR DESCRIPTION
**Pull request by rictic**
I didn't configure renovate right apparently, it should create a single PR to update all the deps

### CODE REVIEW

File: libraries/angular/package.json [hunk 1]
```diff
@@ -23,13 +23,23 @@
...
     "@angular/router": "13.3.1",
+=======
+    "@angular/common": "13.3.2",
+    "@angular/compiler": "13.3.2",
+    "@angular/core": "13.3.2",
+    "@angular/forms": "13.3.2",
+    "@angular/platform-browser": "13.3.2",
+    "@angular/platform-browser-dynamic": "13.3.2",
+    "@angular/router": "13.3.2",
+>>>>>>> Stashed changes
```
Comment (AndrewJakubowicz) [inline libraries/angular/package.json:42]: Accidentally committed markers.

Comment (rictic) [reply libraries/angular/package.json:42]: erp!

**Review by AndrewJakubowicz — CHANGES_REQUESTED** [review]
Nit: Accidental git changes committed. Updated stuff looks good.

File: libraries/angular/package.json (hunk 1 above)
Comment (rictic) [reply libraries/angular/package.json]: Updated

**Review by AndrewJakubowicz — APPROVED** [review]
Nice!

**Pull request merged**